apps/sms/views/conversation/js/conversation.js

```diff
@@ -375,6 +375,34 @@ var ConversationView = {
     this.toggleRecipientSuggestions();
   },
 
+  hasValidRecipients() {
+    if (this.recipients.numbers.length) {
+      return true;
+    }
+
+    var notValidatedNumber = this.recipients.inputValue;
+    return notValidatedNumber && isFinite(notValidatedNumber);
```

Comment:
Btw, this ```isFinite``` allows to send message to ```-1234``` even on master, while our logic marks such number as invalid.